src/librustc_driver/lib.rs

```diff
@@ -114,9 +116,17 @@ pub trait Callbacks {
     }
 }
 
-pub struct DefaultCallbacks;
+#[derive(Default)]
```

Comment:
Create a private copy of this struct instead of changing the public one.

Reply:
Good suggestion. I have uploaded new code that does this.